packages/vow/vat.js

```diff
@@ -36,6 +35,6 @@ export const { watch, when, makeVowKit, allVows } = vowTools;
  * you need to `watch` its resulting promises.
  */
 export const V = makeE(globalThis.HandledPromise, {
```

Comment:
Since you created `heapVowTools`, I would prefer you recategorised `V` as well... it implicitly uses `when`, which may be surprising to people thinking it Just Works until they find out the awful truth that it silently fails if you upgrade the vat that has any in-progress `V` operations.
```suggestion
export const heapVowE = makeE(globalThis.HandledPromise, {...})
```